modules/ossm-rn-new-features.adoc

```diff
@@ -19,7 +19,11 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
-== Component versions included in {SMProductName} version {SMProductVersion}
+== New features {SMProductName} 2.1.2
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
+
+=== Component versions included in {SMProductName} version {SMProductVersion}
```

Comment:
`{SMProductVersion}` is shown is the preview as `2.1.1`. I'm guessing this is going to be fixed automatically, right?

Reply:
Thanks for pointing this out. We started using a new attributes file, which has now been updated. This change has been made.